aspnet/tutorials/mvc-with-entity-framework.rst

```diff
@@ -0,0 +1,394 @@
+
+- `HTML, CSS, JavaScript, SQL, PHP, JQuery <http://www.w3schools.com/>`_
+- `Getting Started with Visual C# <http://msdn.microsoft.com/library/a72418yk.aspx>`_
+- `Relational database concepts <http://en.wikipedia.org/wiki/Relational_database>`_
+- `Multitier architecture concepts <http://en.wikipedia.org/wiki/Multitier_architecture>`_
+- `Cloud development <http://www.asp.net/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/introduction>`_
+
+What you’ll learn
+^^^^^^^^^^^^^^^^^
+The concepts presented in this tutorial include:
+
+- Creating, reviewing and running a new project
+- The file structure of the project
+- How to run the project in Visual Studio 2015
+- The features of the default ASP.NET 5 application
+- How to create a data model
+- How to initialize and seed a database
+- How to configure and update the app to support the database
+- How to publish and run the web app in Azure
+
+In this topic
+^^^^^^^^^^^^^
+- `Introduction`_
+- `Prerequisites`_
+- `Create a new ASP.NET 5 project`_
+- `Entity Framework`_
+- `Create a data model and scaffolding`_
+- `Using data migrations to create the database`_
+- `Adding navigation`_
+- `Run the web app locally`_
+- `Publish the web app to Azure App Service`_
+- `Summary`_
+- `Additional resources`_
+
+Prerequisites
+-------------
+Before you start, make sure that you have the following software installed on your computer:
+
+1. Install `Visual Studio 2015 <http://go.microsoft.com/fwlink/?LinkId=532606>`_
+2. Install the latest `ASP.NET 5 preview (RC 1) <https://docs.asp.net/en/latest/getting-started/installing-on-windows.html>`_
+
+.. note:: For additional information about installing ASP.NET 5 on other platforms, see `ASP.NET 5 Documentation - Getting Started <http://docs.asp.net/en/latest/index.html>`_.
+
+Create a new ASP.NET 5 project
+------------------------------
+
+Start Visual Studio 2015. From the **File** menu, select **New** > **Project**. 
+
+Select the **ASP.NET Web Application** project template. It appears under **Installed** > **Templates** > **Visual C#** > **Web**. Name the project ``ContosoBooks`` and click **OK**.
+
+.. image:: mvc-with-entity-framework/_static/02-new-project.png
+
+In the **New Project** dialog, select **Web Site** under **ASP.NET 5 Preview Templates**.
+
+.. image:: mvc-with-entity-framework/_static/03-web-site-template.png
+
+Running the default app
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Once Visual Studio finishes creating the app, run the app by selecting **Debug** -> **Start Debugging**. As an alternative, you can press **F5**.
+
+It may take time to initialize Visual Studio and the new app. Once it is complete, the browser will show the running app.				
+
+.. image:: mvc-with-entity-framework/_static/05-browser-runapp.png
+
+After reviewing the running Web app, close the browser and click the "Stop Debugging" icon in the toolbar of Visual Studio to stop the app.
+
+Review the project
```

Comment:
I'd delete this section. You only highlight a few files and not in detail. You can point to http://docs.asp.net/projects/mvc/en/latest/migration/migratingfrommvc5.html which I'll have updated in a week and that describes all this in detail.